src/app/Fake.DotNet.Fsdocs/Fsdocs.fs

```diff
@@ -32,12 +32,18 @@ module Fsdocs =
             /// Save images referenced in docs
             SaveImages: bool option
 
-            /// Don't add line numbers, default is to add line number.
-            NoLineNumbers: bool option
+            /// Add line numbers
+            LineNumbers: bool option
```

Comment:
Seems like it's [opt-in for line numbers now ](https://github.com/fsprojects/FSharp.Formatting/blob/e665aa95ab897e5e2c543e5847f3820b3e215e92/src/fsdocs-tool/BuildCommand.fs#L1130)